js/client-chat.js

```diff
@@ -1404,7 +1404,8 @@
 					break;
 
 				case 'error':
-					this.$chat.append('<div class="notice message-error">' + BattleLog.escapeHTML(row.slice(1).join('|')) + '</div>');
+					// eslint-disable-next-line no-undef
+					this.$chat.append('<div class="notice message-error">' + formatText(row.slice(1).join('|'), true) + '</div>');
```

Comment:
is `formatText` not defined here?

Reply:
It works locally. Not sure what's with ESLint.

Reply:
isn't ``formatText`` already used elsewhere in client-chat?

Reply:
Not on a cursory search. The only usage i see is in `battle-log.ts`.

Reply:
aha, looks like you should be using `BattleLog.parseMessage`. You should be adding a `trusted` param _there_.

Reply:
Done.